packages/bundle/src/createCognitiveServicesSpeechServicesPonyfillFactory.spec.js

```diff
@@ -217,6 +55,7 @@ test('not providing reference grammar ID', () => {
 
   ponyfillFactory({});
```

Comment:
should this  be polly?

Reply:
Nope, ponyfill is correct: https://github.com/microsoft/BotFramework-WebChat/issues/2371#issuecomment-526350058